config/settings/development.yml

```diff
@@ -1 +1,4 @@
+symphony_env: '/s/SUL/Config/sirsi.env'
+symphony_host: 'morison.stanford.edu'
 symphony_cgi_url: 'http://morison.stanford.edu/cgi-bin/webforms'
+symphony_catalogdump: '/s/sirsi/Unicorn/Bin/catalogdump -om -kc -h -z -j -n dumpjunktag.Bibframe 2>/dev/null'
```

Comment:
👍 I added a comment to the development.yml file as well as updating the production and stage  yml files in shared_configs:

```
# Note: the dumpjunktag.Bibframe file is located on the Symphony server under /s/sirsi/Unicorn/Custom/.
#       It contains the tag listing '001' which instructs the catalogdump to remove all '001' fields of
#       the MARC record. The -kc flag then adds back in an '001' field with the ckey as the value
```